benchmarks/Neo.Benchmarks/config.json

```diff
@@ -27,8 +27,8 @@
   "ProtocolConfiguration": {
     "Network": 860833102,
     "AddressVersion": 53,
-    "MillisecondsPerBlock": 15000,
-    "MaxTransactionsPerBlock": 512,
+    "MillisecondsPerBlock": 3000,
+    "MaxTransactionsPerBlock": 256,
```

Comment:
MaxTransactionsPerBlock can be kept as is.

Reply:
I think it need to be changed, I think that this comment here contradicts your comment below. I did not understand. You said to cut even more aggressively.

Reply:
We need to reduce `MaxTraceableBlocks`, not `MaxTransactionsPerBlock`.

Reply:
all can be reduced, each block wont be able to process that amount of transactions.

Reply:
C# node can handle several thousands of simple NEP-17 transfers per second. It's all about `MaxBlockSystemFee` (https://github.com/neo-project/neo/issues/3552), not `MaxTransactionsPerBlock`.

Reply:
I think that for a first release these proposed values are good.
They can be increased easily, the opposite is more complicated in my opnion.
So, I think we should just move forward.

Reply:
This actually increase the tps for neo, so 256 is acceptable.